draft-ietf-ppm-dap.md

```diff
@@ -454,13 +454,17 @@ in the "type" field (within the DAP URN namespace
 
 This list is not exhaustive. The server MAY return errors set to a URI other
 than those defined above. Servers MUST NOT use the DAP URN namespace for errors
-not listed in the appropriate IANA registry (see {{urn-space}}). Clients SHOULD
-display the "detail" field of all errors. The "instance" value MUST be the
-endpoint to which the request was targeted. The problem document MUST also
-include a "taskid" member which contains the associated DAP task ID (this value
-is always known, see {{task-configuration}}), encoded in Base 64 using the URL
-and filename safe alphabet with no padding defined in sections 5 and 3.2 of
-{{!RFC4648}}.
+not listed in the appropriate IANA registry (see {{urn-space}}).
+
+For errors defined in this document, the "title" member MUST be set to the
+Description corresponding to the error type in the table above. The "detail"
+member includes additional diagnostic information. The "instance" member is
+typically the endpoint to which the request was targeted.
```

Comment:
@divergentdave is this guidance "instance" useful, but sufficiently ambiguous?

Reply:
I think this is still pointing implementers in the wrong direction compared to RFC 7807. The RFC says that the instance URI "will always identify a specific occurrence of a problem". Thus, the URI would have to include some identifier pointing back to error information from the particular request/trace.

I think it would make sense for DAP to be silent on the `instance` member. It's optional and it's sufficiently specified in RFC 7807 to the extent that DAP doesn't need to add anything further. Plus I have a hunch most DAP implementations won't go to the extent of storing error information such that consumers could dereference `instance` URIs for "further information" as described in RFC 7807 section 3.1.

Reply:
Done.